microsoft/botframework-solutions
# Enable contact search in Email skill

## Description
Enable contact search in Email skill

## Related Issue
Cannot email an specific contact #248

## Testing Steps
1. In your MSA account, add an new contact named "Test" and set email as "test@outlook.com"(Ensure this address never send you an email, otherwise it will be include in people you work with)
2. Open email bot -> "send an email to Test"
3. Test is resolved as contact.
